sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/models.py

```diff
@@ -1143,32 +1105,29 @@ def admin_details(self):
         # type: () -> List[AdministratorDetails]
         """Details of the organization administrator of this issuer.
 
-        :rtype: List[AdministratorDetails]
+        :rtype: List[~azure.keyvault.certificates.models.AdministratorDetails]
```

Comment:
This is a bit subtle, but `List` is a Protocol from `typing` ; `list` is the Python standard class I think we actually return here. `List` makes sense as a type annotation for mypy (though probably `list` would suffice as well). For docstrings I lean toward being more specific, especially when doing so allows us to link to a concrete class whose docs may be useful. That isn't the case here but I can still imagine a reader seizing on the capital L and wondering whether we use some custom `List` class.

Reply:
makes sense, switched to lower case in comments. Thanks!